test/unit/broadcast_active_set.c

```diff
@@ -32,15 +32,36 @@
 
 #define NELEM 10
 
+#ifdef ENABLE_DEPRECATED_TESTS
```

Comment:
Do we have this flag added in configure?

Reply:
There is not yet a configury hook for this, but it's tracked in SOS here:
https://github.com/Sandia-OpenSHMEM/SOS/issues/542

Do you mind if we keep it this way for now and I'll place it towards the top of the v1.5.2 milestone?

Reply:
Sure. That works. Thanks.